packages/tests/src.ts/test-providers.spec.ts

```diff
@@ -882,19 +882,97 @@ testFunctions.push({
-        });
-        assert.ok(!!value);
-    });
-});
+// describe("Test Provider Methods", function() {
```

Comment:
We can remove the commented tests here and add `TODOs` for adding tests for the provider methods once we implement them